learn/features/treasury.md

```diff
@@ -9,66 +9,65 @@ description: As a Polkadot parachain, Moonbeam has an on-chain treasury controll
-    |          Proposal bond           |                            {{ networks.moonbase.treasury.proposal_bond }}% of the proposed spend                            |
-    |      Proposal bond minimum       |                                   {{ networks.moonbase.treasury.proposal_bond_min }} DEV                                    |
-    |      Proposal bond maximum       |                                     {{ networks.moonbase.treasury.proposal_bond_max }}                                      |
-    |         Motion duration          | {{ networks.moonbase.treasury.motion_duration_blocks }} blocks ({{ networks.moonbase.treasury.motion_duration_days }} days) |
-    |           Spend period           |    {{ networks.moonbase.treasury.spend_period_blocks }} blocks ({{ networks.moonbase.treasury.spend_period_days}} days)     |
-    |    Maximum approved proposals    |                                   {{ networks.moonbase.treasury.max_approved_proposals }}                                   |
-    | % of transaction fees allocated  |                                     {{ networks.moonbase.treasury.tx_fees_allocated }}                                      |
+    |                      Variable                      |                                                            Value                                                            |
+    |:--------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------:|
+    |                   Proposal bond                    |             {{ networks.moonbase.treasury.proposal_bond }}% of the proposed spend or the proposal bond minimum              |
+    |               Proposal bond minimum                |                                   {{ networks.moonbase.treasury.proposal_bond_min }} DEV                                    |
+    |               Proposal bond maximum                |                                     {{ networks.moonbase.treasury.proposal_bond_max }}                                      |
+    |                  Motion duration                   | {{ networks.moonbase.treasury.motion_duration_blocks }} blocks ({{ networks.moonbase.treasury.motion_duration_days }} days) |
+    |                    Spend period                    |    {{ networks.moonbase.treasury.spend_period_blocks }} blocks ({{ networks.moonbase.treasury.spend_period_days}} days)     |
+    |             Maximum approved proposals             |                                   {{ networks.moonbase.treasury.max_approved_proposals }}                                   |
+    | Target % of transaction fees allocated to Treasury |                                     {{ networks.moonbase.treasury.tx_fees_allocated }}                                      |
 
 ## Community Treasury {: #community-treasury } 
```

Comment:
Should we include anything around the Forum? There is also the optional IDEA step and the mandatory forum post discussion

Reply:
There was a sentence added in the following paragraph with a link to the forum